controllers/utils/utils.go

```diff
@@ -245,7 +244,8 @@ func VerifyIfMeshAuthorizationIsEnabled(ctx context.Context, cli client.Client)
 	// is available, taking into account both managed an unmanaged configurations.
 
 	authPolicies := &v1beta1.AuthorizationPolicyList{}
-	if err := cli.List(ctx, authPolicies, client.InNamespace(constants.IstioNamespace)); err != nil && !meta.IsNoMatchError(err) {
+	_, meshName := GetIstioControlPlaneName(ctx, cli)
+	if err := cli.List(ctx, authPolicies, client.InNamespace(meshName)); err != nil && !meta.IsNoMatchError(err) {
```

Comment:
you missed some places like this :)

Reply:
sry about asking you to rename, but it it makes things easier to read :)

Reply:
No problem, everything should be updated now :)